src/Illuminate/Auth/Access/Gate.php

```diff
@@ -123,19 +128,27 @@ public function define($ability, $callback)
      * @param  string  $name
      * @param  string  $class
      * @param  array   $abilities
+     * @param  array   $options
      * @return $this
      */
-    public function resource($name, $class, array $abilities = null)
+    public function resource($name, $class, array $abilities = [], array $options = [])
     {
         $abilities = $abilities ?: [
+            'index'   => 'index',
```

Comment:
It feels like you built this feature for yourself, for your own use case, and not for the community general use case.